di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/config/ClientConfig.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.di.ipv.stub.cred.config;
+
+import java.util.Map;
+
+public class ClientConfig {
+    public String getSigningCert() {
+        return signingCert;
+    }
+
+    public Map<String, String> getJwtAuthentication() {
+        return jwtAuthentication;
+    }
+
+    private String signingCert;
```

Comment:
Could we move these vars to the top of the file, think we've always got the getter's/setter methods below the variable elsewhere